broker/message.proto

```diff
@@ -3,13 +3,15 @@
 
 syntax = "proto3";
 package broker;
+import "google/protobuf/timestamp.proto";
 
 // Message represents a message emitted by the Mainflux adapters layer.
 message Message {
-	string channel     = 1;
-	string subtopic    = 2;
-	string publisher   = 3;
-	string protocol    = 4;
-	string contentType = 5;
-	bytes  payload     = 6;
+	string                    channel     = 1;
+	string                    subtopic    = 2;
+	string                    publisher   = 3;
+	string                    protocol    = 4;
+	string                    contentType = 5;
+	bytes                     payload     = 6;
+	google.protobuf.Timestamp created     = 7;
```

Comment:
While I agree with what @drasko said (Timestamp is self-explanatory), I'm not sure that we should use it. Simple timestamp (Unix nanoseconds or milliseconds) is just fine, I think. The benefits of Timestamp are more user-friendly JSON encoding, which translates to a string rather than a number. This can do more harm than good in our case. Also, there is a burden of the imported package. Timestamp itself does not explain the precision, which kinda undoes self-explanatory argument.